local-proxy/src/server.ts

```diff
@@ -14,11 +14,13 @@ import { AddressInfo } from 'net';
 import os from 'os';
 import { initRegistry } from './stdio';
 import nanoid from 'nanoid';
-import mkdirp from 'mkdirp';
-import { copy } from 'fs-extra';
+import { copy, mkdirpSync } from 'fs-extra';
```

Comment:
`mkdirp` can now be removed from `package.json` dependencies.

Similarly possible to change calls to `rimraf`.